### ISSUE TITLE
Fine tuning code might have a small bug (just printing)

### ISSUE THREAD

**Issue by danpf** [agemagician/ProtTrans#34]
Hello,
I hope I'm not wrong and wasting your time. but I think that trainer.predict returns [logits, labels, metrics].
where labels is the labels input into the trainer.  In order to print the actual predicted labels you have to softmax then argmax.
I don't think this has any effect on your metrics at all, just the final parts you are printing at the end

**Comment by agemagician** [comment]
Hi @danpf ,

Good catch, yes, you are right, it only affects the final printing but doesn't affect the performance metrics.
I have updated the fine-tune code to fix this issue.

Thanks a lot 😄
